jablib/src/test/java/org/jabref/logic/layout/LayoutEntryTest.java

```diff
@@ -73,32 +73,38 @@ public String layout(String layoutFile, BibEntry entry) throws IOException {
         return layout.doLayout(entry, null);
     }
 
-    @Test
-    void parseMethodCalls() {
-        assertEquals(1, LayoutEntry.parseMethodsCalls("bla").size());
-        assertEquals("bla", LayoutEntry.parseMethodsCalls("bla").getFirst().getFirst());
-
-        assertEquals(1, LayoutEntry.parseMethodsCalls("bla,").size());
-        assertEquals("bla", LayoutEntry.parseMethodsCalls("bla,").getFirst().getFirst());
-
-        assertEquals(1, LayoutEntry.parseMethodsCalls("_bla.bla.blub,").size());
-        assertEquals("_bla.bla.blub", LayoutEntry.parseMethodsCalls("_bla.bla.blub,").getFirst().getFirst());
-
-        assertEquals(2, LayoutEntry.parseMethodsCalls("bla,foo").size());
-        assertEquals("bla", LayoutEntry.parseMethodsCalls("bla,foo").getFirst().getFirst());
-        assertEquals("foo", LayoutEntry.parseMethodsCalls("bla,foo").get(1).getFirst());
+    @ParameterizedTest
+    @CsvSource(textBlock = """
+            bla, bla
+            bla, bla
+            _bla.bla.blub, _bla.bla.blub
+            """)
+    void parseSingleMethodWithoutArguments(String expected, String input) {
+        assertEquals(1, LayoutEntry.parseMethodsCalls(input).size());
+        assertEquals(expected, LayoutEntry.parseMethodsCalls(input).getFirst().getFirst());
+    }
 
-        assertEquals(2, LayoutEntry.parseMethodsCalls("bla(\"test\"),foo(\"fark\")").size());
-        assertEquals("bla", LayoutEntry.parseMethodsCalls("bla(\"test\"),foo(\"fark\")").getFirst().getFirst());
-        assertEquals("foo", LayoutEntry.parseMethodsCalls("bla(\"test\"),foo(\"fark\")").get(1).getFirst());
-        assertEquals("test", LayoutEntry.parseMethodsCalls("bla(\"test\"),foo(\"fark\")").getFirst().get(1));
-        assertEquals("fark", LayoutEntry.parseMethodsCalls("bla(\"test\"),foo(\"fark\")").get(1).get(1));
+    @Test
+    void parseTwoMethodsWithoutArguments() {
+        String input = "bla,foo";
+        var result = LayoutEntry.parseMethodsCalls(input);
+        assertEquals(2, result.size());
+        assertEquals("bla", result.getFirst().getFirst());
+        assertEquals("foo", result.get(1).getFirst());
+    }
 
-        assertEquals(2, LayoutEntry.parseMethodsCalls("bla(test),foo(fark)").size());
-        assertEquals("bla", LayoutEntry.parseMethodsCalls("bla(test),foo(fark)").getFirst().getFirst());
-        assertEquals("foo", LayoutEntry.parseMethodsCalls("bla(test),foo(fark)").get(1).getFirst());
-        assertEquals("test", LayoutEntry.parseMethodsCalls("bla(test),foo(fark)").getFirst().get(1));
-        assertEquals("fark", LayoutEntry.parseMethodsCalls("bla(test),foo(fark)").get(1).get(1));
+    @ParameterizedTest
+    @CsvSource(delimiterString = ";", textBlock = """
```

Comment:
Test caes in one package should be consistent. Below, you used `,` and `'`. Here, you use `"` and `'`. Please the remove the explicit delimiterString `";"` here and change ";" to "," below. Since you use `'`, it should work.